packages/thirdweb/src/utils/fetch.ts

```diff
@@ -228,6 +228,10 @@ export function getPlatformHeaders() {
     ...(bundleId ? { "x-bundle-id": bundleId } : {}),
   });
 
+  if (typeof window !== "undefined") {
+    previousPlatform.push(["x-sdk-location", window.location.href]);
```

Comment:
this feels a bit odd no? also make sure to handle location undefined as well for some platforms. RN actually can have a window but no location i think

Reply:
How is it odd?

Reply:
can we just add it to the trackPayEvent / trackConnectEvent for now?